tests/toolkit/test_developer.py

```diff
@@ -1,4 +1,18 @@
 from pathlib import Path
+
+def test_system_without_hints(temp_dir, developer_toolkit):
+    (temp_dir / 'prompts').mkdir(parents=True, exist_ok=True)
+    developer_toolkit.workspace = temp_dir
+    assert developer_toolkit.system()
+
+def test_system_with_hints(temp_dir, developer_toolkit):
+    hints_content = "Use Python 3.8"
+    (temp_dir / 'prompts').mkdir(parents=True, exist_ok=True)
+    with (temp_dir / '.goosehints').open('w') as f:
+        f.write(hints_content)
+    developer_toolkit.workspace = temp_dir
```

Comment:
I think this might have been a hallucination? I think instead we need to change dirs to the test dir before loading system